manifests/b/BraveSoftware/BraveBrowser-Beta/94.1.31.58/BraveSoftware.BraveBrowser-Beta.installer.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: BraveSoftware.BraveBrowser-Beta
+PackageVersion: 94.1.31.58
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: user
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: ""
+  SilentWithProgress: ""
```

```suggestion
InstallerSwitches:
  Silent: " "
  SilentWithProgress: " "
```
This would fix
```
Manifest validation succeeded with warnings.
Manifest Warning: Silent and SilentWithProgress switches are not specified for InstallerType exe. Please make sure the installer can run unattended.
```
But YamlCreate removes the blank space, so until we actually find a switch that works there we'll just leave it as ""
None of the known silent switches actually worked there so..